src/secp256k1/src/bench_ecdh.c

```diff
@@ -6,8 +6,8 @@
 
 #include <string.h>
 
-#include "include/secp256k1.h"
```

Comment:
note: paths work when building within our codebase but not the source as it would be installed on the system